tools/liveimage-mount

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/python -tt
+# coding: latin-1
+# 2015-07-23 13:03:05
```

Comment:
We use ``utf-8`` everywhere else in our projects, so this file should use it too. And we don't use the timestamps anywhere, so nobody would ever update it which will result in a misleading information which is worse than no information.